vars/sendTuleapStatus.groovy

```diff
@@ -18,11 +18,11 @@ def call(Map config) {
 
   // Configuration de la step :
   //   gitToken : Token d'accès au dépot GIT
-  //   tuleapServer : chemin vers le server Tuleap (tuleap.net par defaut)
+  //   tuleapServer : chemin vers le server Tuleap
   //   targetRepo : ID ou chemin du dépot
   //   status : success / failure
   def gitToken = config.gitToken
-  def serverPath = config.tuleapServer ?: "https://tuleap.net"
+  def serverPath = config.tuleapServer

```

Comment:
What about throwing an error here, if not set? It could be simpler again, no?

Reply:
Yep, you are right. It would be better.

I will update the PR on Monday.